Source/WebCore/platform/ProcessQualified.h

```diff
@@ -155,4 +155,32 @@ template<typename T> struct HashTraits<WebCore::ProcessQualified<T>> : SimpleCla
     static constexpr bool emptyValueIsZero = HashTraits<T>::emptyValueIsZero;
 };
 
+class ProcessQualifiedStringTypeAdapter {
```

Comment:
In the future someone might want to make this even more generic for if you have a ProcessQualified<T>, but this will work for all ProcessQualified<ObjectIdentifier>'s